package.json

```diff
@@ -55,7 +55,7 @@
   "devDependencies": {
     "@oclif/dev-cli": "^1.22.2",
     "@oclif/test": "^1.2.6",
-    "@twilio/cli-test": "^2.1.0",
+    "@twilio/cli-test": "^2.1.2",
```

Comment:
This bump shouldn't be needed. `^` should handle all the future minor/patch version upgrades with next installation.

Reply:
Yes, Thanks.